test/ETO/ETOCommitment.js

```diff
@@ -2468,12 +2596,12 @@ contract("ETOCommitment", ([, admin, company, nominee, ...investors]) => {
         // we must cross MIN CAP
         if (tokenTermsDict.MIN_NUMBER_OF_TOKENS.gt(totalInvestment[1])) {
           const missingTokens = tokenTermsDict.MIN_NUMBER_OF_TOKENS.sub(totalInvestment[1]);
-          let missingAmount = missingTokens.mul(tokenTermsDict.TOKEN_PRICE_EUR_ULPS);
+          let missingAmount = tokensCostEur(missingTokens, tokenTermsDict.TOKEN_PRICE_EUR_ULPS);
           if (missingAmount.lt(etoTermsDict.MIN_TICKET_EUR_ULPS)) {
             missingAmount = etoTermsDict.MIN_TICKET_EUR_ULPS;
           }
           // console.log(`min cap investment: ${missingTokens} ${missingAmount} EUR`);
-          await investAmount(investors[4], missingAmount, "EUR", tokenprice);
+          await investAmount(investors[4], missingAmount.add(1), "EUR", tokenprice);
```

Comment:
why does it not work without the add(1)? is it because we do some rounding in ```tokenCostEur(...```

i can not let you get away with just adding 1 to make it work :) we need to at least have a theory why

Reply:
good question! it's of course rounding but well understood. this is how price is computed that will let you cross min-cap
```
function tokensCostEur(tokens, price) {
    return tokens.mul(price).divToInt(getTokenPower());
  }
```
so we always round down here with `divToInt` which means that sometimes we'll land 1 wei from crossing the min cap and test fails.